### ISSUE TITLE
Error after start and address problem

### ISSUE THREAD

**Issue by dudek-marcin** [vitalidze/traccar-web#485]
After starting server i have few errors like that: 

WARN: Invalid token=EOF at (line no=1, column no=0, offset=-1). Expected tokens are: [CURLYOPEN, SQUAREOPEN] - JsonParsingException (... < QueryBuilder:361 < *:357 < *:409 < DataManager:367 < ConnectionManager:43 < ...)

What can it be? I have version 3.2 of traccar and your newest web modification

My another problem is lack of address. I tried both Google and Nominatim, but there is null in database and no error in log.

**Comment by vitalidze** [comment]
Try solution from #433

> My another problem is lack of address. I tried both Google and Nominatim, but there is null in database and no error in log.

This sounds like reverse geocoding is not enabled at all.

P.S. have you tried searching over issues before posting your own question?

**Comment by dudek-marcin** [comment]
I used search and I tried solution you have posted (changing :other to :attributes), but I still got that warning after restart, anyway it doesn't have influence on proper working, so its not that important.
I have reverse geodcoding enabled in in config file and also I have tried all 3 providers that are described on official website. The only thing I haven't tried is setting own reverse geocoding service.
